vite-project/src/Components/TicTacToe/TicTacToe.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useRef } from 'react'
+import { useState } from 'react'
+import './TicTacToe.css'
+import circle_icon from '../Assets/circle.jpg'
+import cross_icon from '../Assets/cross.jpg'
+
+let data = ["", "", "", "", "", "", "", "", ""];
+
+const TicTacToe = () => {
+
+    let [count, setCount] = useState(0);
+    let [lock, setLock] = useState(false);
+    let titleRef = useRef(null);
+    let box1 = useRef(null);
+    let box2 = useRef(null);
+    let box3 = useRef(null);
+    let box4 = useRef(null);
+    let box5 = useRef(null);
+    let box6 = useRef(null);
+    let box7 = useRef(null);
+    let box8 = useRef(null);
+    let box9 = useRef(null);
+
+    let box_array = [box1, box2, box3, box4, box5, box6, box7, box8, box9];
+
+    const toggle = (e, num) => {
+        if (lock) {
+            return 0;
```

Comment:
De ce e nevoie de 0?
```suggestion
            return;
```